Assignment2/assignment2.sql

```diff
@@ -1,40 +1,38 @@
 create index idx_batter_counts on baseball.batter_counts(game_id);
 create index idx_game on baseball.game(game_id);
 
+create table baseball.results (batter numeric, annual_batting_average double PRECISION, historic_batting_average double precision, rolling_batting_average double PRECISION);
 
 Question 1
 
 annual
-with t1 as (select btc.team_id, sum(btc.hit)/sum(btc.atBat) as batting_average, extract(YEAR from gm.local_date) as year
+nsert into baseball.results (batter, annual_batting_average) select batter, batting_average from(
+select btc.batter, sum(btc.hit)/sum(btc.atBat) as batting_average, extract(YEAR from gm.local_date) as year
 from baseball.batter_counts btc
 left join baseball.game gm
 on btc.game_id = gm.game_id
-group by (btc.team_id))
```

Comment:
don't group by team.Id group by batter so you can see their average.

Reply:
This was a mistake thanks for letting me know, edited.